constants/index.js

```diff
@@ -0,0 +1,17 @@
+const RESPONSE_MSG = {
+    SUCCESS: "Request Successful.",
+    FAILED: "Something is not good.",
+}
+const RESPONSE_STATUS = {
+    SUCCESS: 0,
+    FAILED: -1
+}
+const STATUS = {
+    SUCCESS: 0,
+    FAILED: -1
+}
+
+const ARRAYLENGTH = 0
+const SAMESCHEME = 1.00.toFixed(2)
+
+module.exports = {ARRAYLENGTH, RESPONSE_MSG, STATUS , SAMESCHEME, RESPONSE_STATUS}
```

Comment:
write it in object format like :
module.exports = {
ARRAYLENGTH,
RESPONSE_MSG, 
STATUS , 
SAMESCHEME, 
RESPONSE_STATUS
}